src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustIndividualValidator.java

```diff
@@ -54,8 +54,8 @@ public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, Strin
                     validateDateBecameInterestedPerson(trustIndividualDto.getDateBecameInterestedPerson(), errors,
                             loggingContext);
                     validateNationality(trustIndividualDto.getNationality(), errors, loggingContext);
-                    validateSecondNationality(trustIndividualDto.getSecondNationality(), loggingContext, errors,
-                            loggingContext);
+                    validateSecondNationality(trustIndividualDto.getNationality(),
```

Comment:
This test does not make sense since  private NationalityValidator nationalityValidator is a mock and you have not set an expectation on it to do the fired test (adding the value to the DTO will not work)